src/pubsub/index.js

```diff
@@ -317,16 +331,18 @@ class PubsubBaseProtocol extends EventEmitter {
 
   /**
    * Responsible for processing each RPC message received by other peers.
-   * @param {string} idB58Str peer id string in base58
-   * @param {DuplexIterableStream} stream inbound stream
-   * @param {PeerStreams} peerStreams PubSub peer
+   *
+   * @param {string} idB58Str - peer id string in base58
+   * @param {DuplexIterableStream} stream - inbound stream
+   * @param {PeerStreams} peerStreams - PubSub peer
    * @returns {Promise<void>}
    */
   async _processMessages (idB58Str, stream, peerStreams) {
     try {
       await pipe(
         stream,
         async (source) => {
+          // @ts-ignore - DuplexIterableStream isn't defined as iterable
```

Comment:
DuplexIterableStream needs to be extracted to types.ts file since its used in multiple places and improved to be an iterable

Reply:
It should really be a muxedStream. Updated it